source/blog/2020-11-11-using-paypal-with-solidus.html.markdown

```diff
@@ -0,0 +1,53 @@
+  PayPal has been working with Solidus in a big way over the last year, find out what's new in the world of payment gateways, and which one makes more sense for your store.
+---
+
+We're excited to share some news with you. As you are probably aware, Solidus has been working with PayPal recently. They've been the gold sponsor for our last two conferences - and more recently, they've helped us revitalize Solidus PayPal Braintree and release the new Solidus PayPal Commerce Platform extension.
+
+We're excited to announce that Solidus PayPal Commerce Platform has just released version 0.2.1, and that Solidus PayPal Braintree has just released version 1.0. Both of these extensions are ready to go, all _you_ need to do is find out which one is right for you.
+
+### Two Extensions, Two Paths
+
+Both extensions bring the magic PayPal buttons into your store, but they operate in very different ways. Below is a quick breakdown of what to expect from each extension.
+
+Before we begin though, it's important to note that both of these extensions use the updated PayPal SDK. This will allow your consumers to checkout using PayPals new financing options, as well as use any payment methods that might be local for them - for instance, your customers from Italy will be able to check out with MyBank, an Italian payment solution.
+
+<img src="/blog/2020/11/11/using-paypal-with-solidus/de.png" />
+
+Both extensions also include the ability to display financing messages to your consumers, which increases overall conversion rates - but that'll be in another blog post! For now, let's focus on the differences between the two extensions.
+
+## Solidus PayPal Braintree
+
+If you've used PayPal with Solidus in the past, this is probably what you're familiar with. Over the last month, we've provided many useful updates to this extension, as well as update the PayPal SDK to work with the newer PayPal options.
+
+If you're already on this extension, the most important changes are that the extension now works with Rails 6, and you can now pass placeholder text or styling instructions to the Braintree hosted fields, making designing around the credit card inputs a bit easier.
+
+So why use this extension over Solidus PayPal Commerce Platform? Well first, it offers more options for your consumers. Through Braintree, you're able to accept credit cards with 3DSecure enabled, and Apple Pay as well. Through hosted fields, you're also given a bit more control over the checkout experience. 
+
+<img src="/blog/2020/11/11/using-paypal-with-solidus/placeholder.png" />
+
+The downside of this extension is that it requires a lot more setup than Solidus PayPal Commerce Platform. The initial setup can take a while, and can be confusing. If you want to use PayPals credit messaging, you'll have to change your settings around in a particular way to allow that. If you want to utilize the cart checkout, you'll need to add the cart checkout partial.
+
+## Solidus PayPal Commerce Platform
```

Comment:
We can probably also say that with SPC we are delegating all the responsibilities around paying with Credit Cards (like fraud detections) to PayPal, while with Braintree should be up to the store owner to take care of that.